### DISCUSSION TITLE
Windows 下使用 MinGW 编译安装 (g)vim 添加 lua 等编程语言支持

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#4] [Announcements]
# Windows 下使用 MinGW 编译安装 (g)vim 添加 lua 等编程语言支持

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/2016/06/14/compile-vim-on-windows-with-lua-support

**Comment by DKSCIcat** [comment]
MinGW已经失效了

**Reply by keelii** [reply]
搜对应的版本下载即可。
